lib/project_types/node/commands/create.rb

```diff
@@ -64,15 +64,19 @@ def self.help
       private
 
       def check_node
-        version, stat = @ctx.capture2e('node', '-v')
-        @ctx.done("node #{version}")
+        output, stat = @ctx.capture2e('which', 'node')
         @ctx.abort(NODE_REQUIRED_NOTICE) unless stat.success?
+
+        version, = @ctx.capture2e('node', '-v')
```

Comment:
Not critical, but if we wanted to be extra cautious we also also check `stat.success?` again here, in case the `node` installation was broken in some way.

Reply:
Don't see why not :), let's do that too.